dev/build_helper_functions.sh

```diff
@@ -137,6 +137,11 @@ function cmake_install {
     "$@"
 
   cmake --build "${BINARY_DIR}"
+  if [ -z "${SUDO}" ]; then
+    if command -v sudo >/dev/null 2>&1 && sudo -n true 2>/dev/null; then
+      SUDO="sudo"
```

Comment:
Can we just let user fix this issue by setting `SUDO=sudo` in the environment?

Reply:
Thank you! I tried in this way. But it is still not working.

Reply:
It seems line 21 should be changed to `SUDO="${SUDO:-""}"`, where all letters are capitalized. Then, I assume setting `SUDO=sudo` in the environment should work.

Reply:
@beliefer, I suggest to just fix this line, and let users explicitly set the environment variable if they meet such permission issue.